src/components/tabs/tabs.js

```diff
@@ -0,0 +1,282 @@
+import '../../vendor/polyfills/Function/prototype/bind'
+import '../../vendor/polyfills/Element/prototype/classList'
+import '../../vendor/polyfills/Event' // addEventListener and event.target normaliziation
+import { nodeListForEach } from '../../common'
+
+function Tabs ($module) {
+  this.$module = $module
+  this.$tabs = $module.querySelectorAll('.govuk-tabs__tab')
```

Comment:
Have we tested having multiple tab components on a single page?

Reply:
Yes, did tested with multiple per page.